pkg/common/cns-lib/volume/manager.go

```diff
@@ -1257,7 +1319,7 @@ func (m *defaultManager) expandVolumeWithImprovedIdempotency(ctx context.Context
 		faultType = ExtractFaultTypeFromVolumeResponseResult(ctx, volumeOperationRes)
 		volumeOperationDetails = createRequestDetails(instanceName, "", "", volumeOperationDetails.Capacity,
 			volumeOperationDetails.OperationDetails.TaskInvocationTimestamp, task.Reference().Value,
-			taskInfo.ActivationId, taskInvocationStatusError, err.Error())
+			taskInfo.ActivationId, taskInvocationStatusError, volumeOperationRes.Fault.LocalizedMessage)
```

Comment:
Just curious - why is this change done?

Reply:
`err` is actually `nil` at this point, so this would have panicked